xdsl/dialects/builtin.py

```diff
@@ -218,6 +220,22 @@ def string_value(self):
         return root
 
 
+class EmptyArrayAttrConstraint(AttrConstraint):
```

Comment:
Could you please add some documentation strings for this class, and the two values below?